lib/validation.ts

```diff
@@ -168,20 +168,17 @@ export function validateCmsPublishMode(
 }
 
 export function fileExists(_path: string): boolean {
```

Comment:
I made some edits to this so I could use it in my new commands. I didn't want to include the logs. Nothing was calling this function except for `checkAndConvertToJson` so this should be a safe change. I tested it on the commands that use `checkAndConvertToJson` and everything seems to WAD